src/NUnitFramework/framework/Constraints/Comparers/ComparisonState.cs

```diff
@@ -9,7 +10,7 @@ namespace NUnit.Framework.Constraints.Comparers
         /// <summary>
         /// Flag indicating whether or not this is the top level comparison.
         /// </summary>
-        public readonly bool TopLevelComparison { get; }
+        public bool TopLevelComparison { get; }
```

Comment:
Just to be sure - I haven't worked much with `readonly ref structs`, `readonly structs`, nor `ref structs` - these are removed as they are redundant due to the `readonly` on the class?

Reply:
yep. that’s correct